package.json

```diff
@@ -1,58 +1,32 @@
 {
-  "name": "nativescript-pdf-view",
-  "description": "A NativeScript plugin to display PDF files on iOS and Android",
-  "main": "src/plugin.js",
-  "version": "1.2.1",
+  "name": "ns-pdfviewer",
```

Comment:
think the old name is good, but then we need to come up with a beter package name for example`nativescript-pdf-viewer-xcode-8`